src/plugins/intel_cpu/src/utils/general_utils.h

```diff
@@ -53,6 +59,15 @@ constexpr bool implication(bool cause, bool cond) {
     return !cause || !!cond;
 }
 
+template <typename T, typename... Ts>
+constexpr bool one_of_v = (std::is_same_v<T, Ts> || ...);
+
+template <typename T, typename... Ts>
+constexpr bool none_of_v = !one_of_v<T, Ts...>;
+
+template <typename T, typename... Ts>
+constexpr bool all_of_v = (std::is_same_v<T, Ts> && ...);
```

Comment:
It should be same as using `std::conjunction` ?

Reply:
Yes, but conjunction is to 'mathy' name for such a simple idea.
Even in cpp reference they provide examples which are just similar wrappers.
I will think about whether we can simplify it more.